metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/RestFunctionsTest.java

```diff
@@ -602,4 +606,27 @@ public void restGetShouldCloseHttpClient() throws Exception {
     verifyNoMoreInteractions(httpClient);
   }
 
+  @Test
+  public void restGetShouldParseResponse() throws Exception {
```

Comment:
Do we want these to be separate tests? it'd be nice to know exactly what each failure is if we ever have them in the future.

Reply:
No problem.  Done.